controllers/spec/common.go

```diff
@@ -511,6 +516,10 @@ func getLegacyDownloadCommand(downloadPath, componentPackage string, authProvide
 func getDownloadCommand(downloadPath, componentPackage string, tlsProvided, authProvided bool, tlsConfig TLSConfig,
 	authConfig *v1alpha1.AuthConfig) []string {
 	var args []string
+	if hasHTTPPrefix(downloadPath) {
+		args = append(args, "wget", downloadPath, "-O", componentPackage)
```

Comment:
do we need to check the availability of `wget` first, then call it to download?

Reply:
the `pulsarctl` image has `wget`, I think we can update the doc to inform users what a `downloadContainer` requires (currently, it requires pulsarctl and wget)